src/Adafruit_SPIFlashBase.cpp

```diff
@@ -57,33 +57,49 @@ Adafruit_SPIFlashBase::Adafruit_SPIFlashBase(
   _flash_dev = NULL;
 }
 
-bool Adafruit_SPIFlashBase::begin(SPIFlash_Device_t const *flash_dev) {
+static SPIFlash_Device_t const *findDevice(SPIFlash_Device_t const *device_list,
+                                           int count,
+                                           uint8_t const (&jedec_ids)[3]) {
```

Comment:
please change the 3rd parameter to `uint8_t const jedec_ids[3]` it is much cleaner than reference to array.

Reply:
Ah, I didn't know that C++ could catch this at compile time. We learn new thing everyday. This is OK to keep this then.